server/lib/mail/templates/password-change.txt

```diff
@@ -0,0 +1,15 @@
+******************************
+ShieldBattery Password Changed
+******************************
+
+You are receiving this email because
```

Comment:
This sounds like our system is inherently insecure, would probably phrase differently. Maybe something like:
`You are receiving this email because the password for {{username}} on ShieldBattery has been changed. If you did not change it, please contact the ShieldBattery team.`

We may also want to direct them to the password reset page rather than having them contact us, I feel like that's probably where we'd end up sending them anyway.